meson.build

```diff
@@ -133,7 +133,7 @@ install_headers(files(
 ## Build library
 inc = [ include_directories('include', 'include/taskomat/sol', 'src') ]
 
-gul_dep = dependency('libgul14')
+gul_dep = dependency('libgul14', version : '> 2.6', fallback : ['libgul14', 'libgul_dep'])
```

Comment:
Ah, this wrap was better than the one we (I) pulled to `main` (i.e. contains version).
So maybe keep this commit.

Reply:
Ok.